src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -22,7 +22,7 @@
       suite('<%= elementName %>', function() {
         test('instantiating the element works', function() {
           var element = fixture('basic');
-          assert.equal(element.is, '<%= elementName %>');
+          assert.equal(element.constructor.is, '<%= elementName %>');
```

Comment:
I would be concerned about putting a number of obtuse tests that might be hard to understand, given that this should really be an example for people (in reality they probably won't be testing the element name and will actually just be testing that it behaves as they expect)